src/mono/mono/mini/mini-amd64.h

```diff
@@ -459,6 +459,8 @@ typedef struct {
 #define MONO_ARCH_HAVE_INTERP_PINVOKE_TRAMP
 #define MONO_ARCH_HAVE_INTERP_ENTRY_TRAMPOLINE 1
 #define MONO_ARCH_HAVE_INTERP_NATIVE_TO_MANAGED 1
+// FIXME: Doesn't work on windows
+//#define MONO_ARCH_HAVE_INIT_MRGCTX 1
```

Comment:
Why?

Reply:
Causes test failures on CI if enabled.

Reply:
That's kind of surprising.  Is the scratch register clobbering some real values for some reason? Or some calling convention problem?

Reply:
Couldn't reproduce locally, so just disabled it for now.